src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -42,25 +43,21 @@ public class Swerve extends GBSubsystem {
     private final SwerveState currentState;
     private Supplier<Rotation2d> currentAngleSupplier;
 
-    public Swerve() {
+    public Swerve(RobotType robotType) {
         setName(getClass().getSimpleName());
 
         this.currentState = new SwerveState(SwerveState.DEFAULT_DRIVE);
-        this.modules = getModules();
-        this.gyro = SwerveGyroFactory.createSwerveGyro();
+        this.modules = new Module[]{
```

Comment:
Why not get modules

Reply:
because then i can do oopsi poopsi and use getModules not in constructur and then code broke

Reply:
K